simulation/tests/workflow_enablement/test_demo_script.py

```
@@ -0,0 +1,162 @@
+
+SIMULATION_ROOT = Path(__file__).resolve().parents[2]
+SCRIPT = SIMULATION_ROOT / "scripts" / "pilot_site_workflow_enablement_demo.py"
+DISCLAIMER = "SIMULATED PILOT SCENARIO — NOT LIVE CUSTOMER DATA"
+RANGE_OPS_ID = "range.closed_loop_collection_handoff"
+
+
+def run_demo(out_dir: Path, *, hash_seed: str | None = None):
+    environment = dict(os.environ)
+    if hash_seed is not None:
+        environment["PYTHONHASHSEED"] = hash_seed
+    return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)